File: 2/1_Cuatrimestre/Programacion_y_Diseño_Orientado_a_Objetos/JavaFinal/NapakalakiJavaSinInterfaz/src/NapakalakiGame/Prize.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package NapakalakiGame;

/**
 *
 * @author javigomez
 */
public class Prize {
    private int treasures;
    private int level;
    
    public Prize(int t, int l){
        treasures=t;
        level=l;
    }
    public int getTreasures(){
        return treasures;
    }
    public int getLevel(){
        return level;
    }
    public String toString(){
        return "level->"+level+" treasures->"+treasures;
    }
}

File: 2/1_Cuatrimestre/PDOO/RubyFinal/NapakalakiRuby/pkg/NapakalakiRuby-0.0.1/lib/Prize.rb
# To change this license header, choose License Headers in Project Properties.
# To change this template file, choose Tools | Templates
# and open the template in the editor.
module NapakalakiGame
class Prize
  attr_accessor :levels, :treasures
  
  def initialize(t,l)
    @levels = l
    @treasures = t
  end
 end
end

